### ISSUE TITLE
Feature: GraceMemer (pun on the name for the bot "DankMemer"

### ISSUE THREAD

**Issue by Gangsta-Muffin** [Code-Society-Lab/grace#98]
**Describe the feature**

A command, that allows to make quick and easy memes out of templates - perhaps predefined templates (more on this below)

How it would work:

A slash command with slash options that represent different meme formats. For example the drake yes/no meme, darth kermit meme, etc etc. When running the command, you can use text Inputs to put text into the typical positions, where they would usually be in a meme. (view images below). 

What I mean by predefined templates:
Basically, some of the text boxes would already be filled out. For example, if we take the picture below, the text `Work on Cursif` and `Malassi` would already be on the image, and then the user can just provide an input for where it sais `Inubakkari`

There are various different python libraries available to do this :)

**Additional context** 
 
![image](https://user-images.githubusercontent.com/94480512/222984551-81630aa5-bc12-456b-a4e4-ccabadec7861.png)
